doc/yjit/yjit.md

```diff
@@ -51,39 +51,55 @@ Because there is no GC for generated code yet, your software could run out of ex
 
 ## Installation
 
-Current YJIT versions are installed by default with CRuby. Make sure to specify the "--yjit" command line option to enable it at runtime.
+### Requirements
 
-Experimental YJIT patches that have not yet been merged with CRuby can be found in ruby-build:
+You will need to install:
+- A C compiler such as GCC or Clang
+- GNU Make and Autoconf
+- The Rust compiler `rustc` and Cargo (if you want to build in dev/debug mode)
```

Comment:
Since 2021 edition is required, we should specify that. Not sure how up to date rustc is for the various package managers

```suggestion
- The Rust compiler `rustc` (>= 1.56.0
) and Cargo (if you want to build in dev/debug mode)
```